applications/ContactStructuralMechanicsApplication/tests/test_mortar_mapper.py

```diff
@@ -0,0 +1,218 @@
+        #out.ExecuteFinalizeSolutionStep()
+
+        ## DEBUG
+        #self.__post_process()
+                    
+    def test_basic_mortar_mapping_triangle(self):
+        input_filename = os.path.dirname(os.path.realpath(__file__)) + "/integration_tests/test_integration_triangle"
+        self._mapper_tests(input_filename, 3)
+        
+    def test_less_basic_mortar_mapping_triangle(self):
+        input_filename = os.path.dirname(os.path.realpath(__file__)) + "/integration_tests/test_integration_triangles"
+        self._mapper_tests(input_filename, 3)
+        
+    def test_simple_curvature_mortar_mapping_triangle(self):
+        input_filename = os.path.dirname(os.path.realpath(__file__)) + "/integration_tests/test_simple_curvature"
+        self._mapper_tests(input_filename, 3)
+        
+    def test_mortar_mapping_triangle(self):
+        input_filename = os.path.dirname(os.path.realpath(__file__)) + "/integration_tests/test_double_curvature_integration_triangle"
+        self._mapper_tests(input_filename, 3)
+        
+    def test_mortar_mapping_quad(self):
+        input_filename = os.path.dirname(os.path.realpath(__file__)) + "/integration_tests/test_double_curvature_integration_quadrilateral"
+        self._mapper_tests(input_filename, 4)
+        
+    def __post_process(self):
+        from gid_output_process import GiDOutputProcess
+        self.gid_output = GiDOutputProcess(self.main_model_part,
+                                    "gid_output",
+                                    KratosMultiphysics.Parameters("""
+                                        {
+                                            "result_file_configuration" : {
+                                                "gidpost_flags": {
+                                                    "GiDPostMode": "GiD_PostBinary",
+                                                    "WriteDeformedMeshFlag": "WriteUndeformed",
+                                                    "WriteConditionsFlag": "WriteElementsOnly",
+                                                    "MultiFileFlag": "SingleFile"
+                                                },        
+                                                "nodal_results"       : ["VELOCITY","PRESSURE"],
+                                                "nodal_nonhistorical_results": ["NORMAL","NODAL_AREA"],
+                                                "nodal_flags_results": ["MASTER","SLAVE"]
+                                            }
+                                        }
+                                        """)
+                                    )
+
+        self.gid_output.ExecuteInitialize()
+        self.gid_output.ExecuteBeforeSolutionLoop()
+        self.gid_output.ExecuteInitializeSolutionStep()
+        self.gid_output.PrintOutput()
+        self.gid_output.ExecuteFinalizeSolutionStep()
+        self.gid_output.ExecuteFinalize()
+
+    def __sci_str(self, x):
```

Comment:
maybe a small comment what this is doing

Reply:
It is just to create an ouput in scientific notattion , right now not used

Reply:
hm so from `0.0258` to `2.58E-2`?
Is there no function for doing this in Python?
If not then I might save that for late use :D